code/modules/mob/living/simple_animal/slime/slime_mob.dm

```diff
@@ -409,7 +409,7 @@
 		new /obj/effect/temp_visual/heart(loc)
 		visible_message("<span class='notice'>[user] feeds the slime some plasma. It chirps happily!</span>", "<span class='notice'>[user] feeds you a few sheets of plasma! Yummy!!!</span>")
 		S.use(5)
-		if(Discipline)
+		if(Discipline && !trained)
```

Comment:
actually, theres an else check below here. Put this !trained in another if so it doesnt fall into the else check